lang/fr/lang.php

```diff
@@ -169,15 +169,53 @@
             'cookies'     => 'Cookies',
         ],
     ],
     'cookie'          => [
         'fields' => [
-            'code'          => 'Code',
-            'intial_status' => 'Ce cookie est activé par défaut',
-            'levels'        => 'Niveaux de cookie',
+            'name_owner'                  => 'Owner',
+            'name_owner_comment'          => 'Owner Name of the Service e.g. Google Analytics',
+            'code'                        => 'Code',
+            'code_comment'                => 'This is generated for the plugin to work (you dont need to touch this)',
+            'intial_status'               => 'This cookie is enabled by default',
+            'levels'                      => 'Cookie levels',
+            'purpose'                     => 'Purpose',
+            'purpose_comment'             => 'Under ePrivacy Law (Article 4, para 11 and article 13, para 1, C and para 2, F), you need to Describe the Purpose of the Use, Processing and any Notifications e.g. \'These cookies are used from AddThis social sharing widget in order to make sure you see the updated count when you share a page\'.',
+            'contact_details_dpo'         => 'Contact Details of Data Controller',
+            'contact_details_dpo_comment' => 'Under ePrivacy Law (Article 13, Para 1, A and Recital 42), you have to give the DPO Contact Address e.g. Cloudflare, Inc. 101 Townsend St. San Francisco, CA 94107. Attention: Data Protection Officer, privacyquestions@cloudflare.com (please note the email address is optional)',
+            'contact_links_dpo'           => 'Contact Links of Data Controller',
+            'contact_links_dpo_comment'   => 'Under ePrivacy Law (Article 13, para 2, B, C and D), you need to give links for \'rights to complain to Supervisory authority and to access correct data policy\' this means linking to their Privacy Policy, Terms and Conditions, Cookie Privacy and Opt-Out Pages if they have them.',
+            'contact_links_dpo_prompt'    => 'Add new link',            
+            'contact_links' => [
+                'name' => 'Anchor Text',
+                'href' => 'Link Target (absolute URL)',
+            ],
+            'cookie_levels' => [
+                'purpose_prompt'     => 'Add Granular Purpose Levels',
+                'purpose_label'      => 'Granular Purpose Levels',
+                'section1_label'     => 'Helpful Tip for Granular Purpose Levels',
+                'section1_comment'   => 'Under ePrivacy Law (Recital 32 and Article 29 working party guidance nov 2017), Granular Opt-In for several Processes. This means having an off setting, the having a basic setting, then having a more advanced setting and so on up to a full setting. An example of this could be Google Analytics where we have an Off Setting, then a basic Tracking Setting, then a basic Tracking Setting plus Plugins, then finally a full setting with everything. The number of levels depends on what you are adding.',
```

Comment:
> Under ePrivacy Law (Recital 32 and Article 29 working party guidance nov 2017), Granular Opt-In for several Processes.

Maybe it's my english, but this doesn't sound like a complete sentence to me?

Reply:
"It's lawyer / patent speak", better grammar would be:

> Under ePrivacy Law (Recital 32 and Article 29 working party guidance nov 2017), Use a Granular Opt-In process with our manage levels feature.

Reply:
p.s. could you change the english lang file to that. If you think that's a better sentence.

Reply:
Thank you for clarifying that!